tests/sqllogictests/suites/mode/standalone/explain/join_reorder/star_multiple_tables.test

```diff
@@ -40,6 +40,8 @@ analyze table t4
 statement ok
 analyze table t5
 
+# A flaky test, analyze table may get different results.
```

Comment:
After https://github.com/datafuselabs/databend/pull/9640, we can retry the test @leiysky